app/main/index.ts

```diff
@@ -294,24 +294,39 @@ app.on('ready', () => {
 	ipcMain.on('downloadFile', (_event: Electron.IpcMessageEvent, url: string, downloadPath: string) => {
 		page.downloadURL(url);
 		page.session.once('will-download', (_event: Event, item) => {
-			const filePath = path.join(downloadPath, item.getFilename());
-
-			const getTimeStamp = (): any => {
-				const date = new Date();
-				return date.getTime();
-			};
-
-			const formatFile = (filePath: string): string => {
-				const fileExtension = path.extname(filePath);
-				const baseName = path.basename(filePath, fileExtension);
-				return `${baseName}-${getTimeStamp()}${fileExtension}`;
-			};
-
-			// Update the name and path of the file if it already exists
-
-			const updatedFilePath = path.join(downloadPath, formatFile(filePath));
-
-			const setFilePath = fs.existsSync(filePath) ? updatedFilePath : filePath;
+			let setFilePath: string;
+			let shortFileName: string;
+			if (ConfigUtil.getConfigItem("promptDownload", false)) {
```

Comment:
@akashnimare I just noticed that the `quotes` field for `xo` overrides is set to 0. I'm guessing it's been done so we can use escape strings as well, but it might lead to inconsistency when writing strings. Do you think it's a good idea to set up a post-commit prettier hook?